Scripts/BrightSign_Scripts/QuickStart-autorun.brs

```diff
@@ -0,0 +1,33 @@
+' How to use:
+' Set the width and height to match your monitor resolution
+' Rotate the content to portrait by uncommenting the "transform" if required
+' Rename the file to "autorun.brs"
+' Copy to the root of the SD card along with the "Tooling-Package"
+
+Sub Main()
+	x = 0
+	y = 0
+	width = 1920
+	height = 1080
+	rect = CreateObject("roRectangle", x, y, width, height)
+	
+	url = "file:/SD:/Tooling_Package/quick-start/Quick-Start_Example.html"
+	
```

Comment:
This seems to want to render only the corner of the Quick Start example on a 2560x1440 monitor, I have tried:

- width+Height set to 1920x1080
- width+Height set to 2560x1440

I get the same visual result with both settings

Reply:
Setting the resolution in the autorun.brs file to 1280x720 foxes this issue

I suggest we add a comment explaining that as-well-as change the default to it

Reply:
Have added a comment to explain to use 1280x720. 
Have not changed the default as this causes the content to draw in a window on a 1080 display